requirements/base.txt

```diff
@@ -12,3 +12,5 @@ edx-opaque-keys==0.2.1
 edx-rest-api-client==1.6.0
 ddt==1.1.1
 mock==1.0.1
+temp-mail==0.2
+python-guerrillamail==0.2.0
```

Comment:
* This requirement was causing issues for me during installation on my local devstack. 
  * I wasn't really sure how it is actually working correctly on jenkins/travis. 
  * I suspect that in those cases the systems were installing from wheels so it worked. 
* I submitted [an upstream PR](https://github.com/ncjones/python-guerrillamail/pull/9) with a fix. We can use that if we need.
* I was also able to work around locally by installing with 
  * `pip install --use-wheel python-guerrillamail==0.2.0` 
  * and then `pip install -r requirements/base.txt`